src/main/java/org/example/factoring/ReadingTheGame.java

```diff
@@ -0,0 +1,49 @@
+package org.example.factoring;
+
+
+import org.example.arraysCreating.Paair;
+import org.example.arraysCreating.SaveTheGame;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public  class ReadingTheGame {
```

Comment:
ja bym to nazwal SaveTheGameDAO, po klasie reading nie domyslil bym sie ze to ma interakcje z baza danych :) 
BTW masz nadmiarowa spacje pomiedzy publi i class :)